Assets/Scripts/Utilities/Statics.cs

```diff
@@ -7,6 +7,13 @@ public static class Statics
     private static ScoreModel scoreModel;
     private static WaveIndicatorView waveIndicator;
 
+    public static GameMode gameMode = GameMode.SINGLEPLAYER;
+    
+    public enum GameMode {
+        SINGLEPLAYER,
+        LOCAL_MULTIPLAYER
+    }

```

Comment:
Hmm, why as nested class?

Reply:
yea good point I will refactor this I think